packages/shorebird_cli/lib/src/extensions/iterable.dart

```diff
@@ -0,0 +1,12 @@
+/// Provides the [containsAnyOf] method to all [Iterable]s.
+extension ContainsAnyOf<T> on Iterable<T> {
+  /// Returns `true` if any of the elements in [elements] are in this iterable.
+  bool containsAnyOf(Iterable<T> elements) {
+    for (final element in elements) {
+      if (contains(element)) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
I was very surprised to not find something in `package:collection` for this. If you're aware of something similar that I missed, please lmk

Reply:
This is just `elements.any((e) => iterable.contains(e))`, right?  It's O(N*M) in worst case.